packages/metaconfig/versions/0.17.x.js

```diff
@@ -13,6 +15,18 @@ class ZeroSeventeen {
       this.kind = 'service'
     }
   }
+
+  up () {
+    const original = this.config
+    const config = rfdc(original)
+    config.$schema = 'https://platformatic.dev/schemas/v0.18.0/db'
+    config.db = config.core
+    delete config.core
```

Comment:
I guess it doesn't matter, but shouldn't this only happen when the `kind` is `'db'`?

Reply:
good spot, fixed.